packages/maskbook/src/components/InjectedComponents/PayloadReplacer.tsx

```diff
@@ -27,7 +27,11 @@ export function PayloadReplacer({ payload }: PayloadReplacerProps) {
                         node.style.setProperty('color', theme.palette.primary.main, 'important')
                     }
                 }}
-                href={payload.startsWith('https://maskbook.com') ? payload : 'https://maskbook.com'}>
+                href={
+                    payload.startsWith('https://mask.io') || payload.startsWith('https://maskbook.com')
+                        ? payload
+                        : 'https://mask.io'
+                }>
```

Comment:
```suggestion
                href={/^https?:\/\/mask(\.io|book\.com)/i.test(payload) ? payload : 'https://mask.io'}>
```

Reply:
No, I don't find any good reason to prefer regexp. It's harder to read and harder to maintain. If you're not handling a very flexible string match, you don't need it.

Reply:
```
http://mask.io
https://mask.io
http://maskbook.com
https://maskbook.com
```

Reply:
I think we have never used the HTTP version.

Reply:
Do you feel OK with my response? If so I will merge this untouched. @septs